setup.py

```diff
@@ -29,30 +28,9 @@
 extras_require = {
     "seeedstudio": ["pyserial>=3.0"],
     "serial": ["pyserial~=3.0"],
-    "neovi": ["python-ics>=2.12", "filelock"],
+    "neovi": ["python-ics>=2.12"],
```

Comment:
Should this change be here?

Reply:
Not necessarily, but i noticed that filelock is mentioned in install_requires already.